v22.2/add-column.md

```diff
@@ -11,6 +11,12 @@ docs_area: reference.sql
 
 {% include {{ page.version.version }}/sql/combine-alter-table-commands.md %}
 
+This is a new sentence.
+
+Bababa testtest
+test test test test test test this should fail repetition asdflkmakl;m4w3;lraksfmdl;xcvkn;4uitnu3warksfljdfndlasjkfnlakjwrnalfkjsndkljn
```

Comment:
This seems non-ideal to have a separate failure for each repetition.

Reply:
I can possibly tweak the RegEx to catch one or more repetitions.

Reply:
Found a better RegEx.